### PR TITLE
WIP authors: uuid generator

### PR DESCRIPTION
**Pull request by jacenkow**
* New method, which generates uuid of authors in HEP, on record-insert.

Signed-off-by: Grzegorz Jacenków <grzegorz.jacenkow@cern.ch>

### CODE REVIEW

File: inspire/modules/authors/receivers.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+
+import uuid
+
+from invenio_records.signals import before_record_insert
+
+
+@before_record_insert.connect
+def assignUuid(sender, *args, **kwargs):
```
Comment (jmartinm) [inline inspire/modules/authors/receivers.py]: please use `assignuuid` as name of the function as Python convention is not camel case.

**Comment by jmartinm** [comment]
Looks good @jacenkow 

Can you have a look at this signal https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/signals.py#L67 so that we also add `uuids` when a record is updated.

**Comment by jacenkow** [comment]
Fixed @jmartinm

**Comment by jmartinm** [comment]
Tested and merged as 4cb98cce0f5988a22a59c27c4373fd7af4c3f682

**Comment by jmartinm** [comment]
Note that I amended the commit message so that the description does not go over 70 lines

**Pull request closed without merging**